lilypond/songs/love-divine-all-loves-excelling/love-divine-all-loves-excelling.ly

```diff
@@ -0,0 +1,55 @@
+\version "2.22.1"
+#(ly:set-option 'relative-includes #t)
+\include "../../lib/hymn-common.ily"
+
+%% TUNE INFO
+\include "../../shared-tunes/hyfrydol-public.ily"
+quarternoteTempo = 120
+
+%% SONG INFO
+title = \titleText "Love Divine, All Loves Excelling"
+poet = \smallText "Text: Charles Wesley, 1747"
+typesetter = "Zachary Cline"
+verseCount = 4
+tags = "english christian 4part"
+dateAdded = "2022-05-12"
+\include "../../lib/header.ily"
+
+%% LYRICS
+verseA = \lyricmode {
+  \l Love di -- vine, all loves ex -- cel -- ling, Joy of heaven to earth come down,
+  \l Fix in us Thy hum -- ble dwell -- ing, All Thy faith -- ful mer -- cies crown!
+  \l Je -- sus, Thou art all com -- pas -- sion, Pure, un -- bound -- ed love Thou art;
```

Comment:
...So regarding capitalization in the lyrics, I think modern hymnals are pretty split, and I think I have a good argument for more "prose-like" capitalization in hymns:

References to God in hymns are usually capitalized, even when metaphorical, and the capitalization helps call attention to that. For example, I think "Joy" = Jesus = God in the above, and that's why it's capitalized. But I could be wrong.

We get all this ambiguity when it coincides with the structural capitalization of the poetic line. I'm guessing "Pure" is just structural capitalization, but it leaves some ambiguity in there.

If you really prefer it this way, I'm not going to force you to change it, though.

Reply:
I tried to change the capitalization in both hymns to match the style of others, but let me know if there's anything else that needs fixing.